src/OpenLoco/src/Objects/TreeObject.cpp

```diff
@@ -195,10 +195,15 @@ namespace OpenLoco
                 sprites[1][variant] = sprites[0][variant] + numPrimaryImages;
             }
 
-            shadowImageOffset = numPrimaryImages * 2;
+            nextImageOffset = numPrimaryImages * 2;
         }
         else
-            shadowImageOffset = numPrimaryImages;
+            nextImageOffset = numPrimaryImages;
```

Comment:
can you put this behind {}. It just looks odd having one branch {}.

Reply:
```suggestion
        {
            nextImageOffset = numPrimaryImages;
        }
```